dotcom-rendering/src/web/lib/decideContainerOverrides.ts

```diff
@@ -19,6 +21,8 @@ const textCardHeadline = (containerPalette: DCRContainerPalette): string => {
 			return '#041F4A';
 		case 'EventAltPalette':
 			return '#041F4A';
+		case 'SpecialReportAltPalette':
+			return '#2b2b2a';
```

Comment:
I went with hex here for consistency but wondered why we're not using the source palette references?

Reply:
I believe it’s because there is no source value for that specific colour.

Reply:
I am keen to attempt replacing these hex values with palette references at some point but indeed I don't know if they all exist in source off the top of my head. Will give it a go at some point and we'll see how that goes!